proj/src/main/webapp/script.js

```diff
@@ -428,6 +428,35 @@ function addUsernameToMessage(uid, elementId) {
     })
 }
 
+const bioBox = document.getElementById('user-bio');
+const editInputBox = document.getElementById('bio-edit');
```

Comment:
Is this and the functions below wrapped in some init function or are they just in the script. It's hard to tell with this diff tool. I think it would be better to put them in an onInit type function rather than defining constants in the middle of the script. Or defining all constants at the top is an alternative solution so they are all visible and don't get lost in the code.

Reply:
fixed@